### PR TITLE
Feature/is 210/header styles favourites counter

### PR DESCRIPTION
**Pull request by iakivpekarskyi**
1. Added new styles to the "cart" and "favourites"
2. Added shadow to header.
3. Added counter to "favourites"
4. Replaced "favourites" and "cart" icons.
5. Added margin to layout so that content doesn't stick to the screen borders

### CODE REVIEW

**Review by Lightness322 — DISMISSED** [review]
Approved

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -8,22 +8,21 @@ import HeaderHeart from '../UI/HeaderHeart/HeaderHeart'
...
     >
       <Link href="/">
-        <div className={' flex items-center gap-4'}>
-          <div className={'h-[21px] w-[19px] sm:h-[31px] sm:w-[28px]'}>
+        <div className=" flex items-center gap-4">
+          <div className="h-[21px] w-[19px] sm:h-[31px] sm:w-[28px]">
             <Image src={logo} width={28} alt="Logo" priority />
           </div>
-          <span className={'hidden items-center text-L sm:flex'}>IL</span>
+          <span className="hidden items-center text-L sm:flex">Iced Latte</span>
```
Comment (anksuunamun) [inline src/components/Header/Header.tsx:19]: На макете - IL.

**Review by anksuunamun — CHANGES_REQUESTED** [review]
После изменений в ПР футер на странице корзины расположен неверно:
image

File: public/heart_black.svg [hunk 2]
```diff
@@ -1,3 +0,0 @@
-<svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply public/heart_black.svg:1]: designers said it will be always grey.

File: README.md [hunk 3]
```diff
@@ -165,6 +167,7 @@ Take some time to press F and give some respects to our [best contributors](http
 - [@nnick44](https://github.com/nnick44)
 
 #### 🖥️ Frontend developers
+
 - [@iakivpekarskyi](https://github.com/iakivpekarskyi)
 - [@anksuunamun-govorov](https://github.com/anksuunamun)
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply README.md:172]: Wasn't me but fixed it.

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (iakivpekarskyi) [reply src/components/Header/Header.tsx:19]: the dot is excessive.

**Comment by iakivpekarskyi** [comment]
> После изменений в ПР футер на странице корзины расположен неверно: image

fixed.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (anksuunamun) [reply README.md]: Thanks!)))

**Review by anksuunamun — APPROVED** [review]

**Pull request merged**